pkg/provider/aws/knowledgebase/edges.go

```diff
@@ -1,18 +1,19 @@
-package edges
+package knowledgebase
 
 import (
 	"fmt"
 	"reflect"
 
 	"github.com/klothoplatform/klotho/pkg/core"
+	knowledgebase "github.com/klothoplatform/klotho/pkg/knowledge_base"
 	"github.com/klothoplatform/klotho/pkg/provider/aws/resources"
 )
 
-var KnowledgeBase = core.EdgeKB{
-	core.Edge{From: reflect.TypeOf(&resources.LambdaFunction{}), To: reflect.TypeOf(&resources.RdsInstance{})}: core.EdgeDetails{
-		ExpansionFunc: func(from, to core.Resource, dag *core.ResourceGraph, data core.EdgeData) error {
-			lambda := from.(*resources.LambdaFunction)
-			instance := to.(*resources.RdsInstance)
+var AwsKB = knowledgebase.EdgeKB{
```

Comment:
For convenience (not sure how others feel about this), we could alias some of the commonly used types - especially because the package name is the same.
ie:
```go
type (
    EdgeKB = knowledgebase.EdgeKB
    Edge = knowledgebase.Edge
    EdgeDetails = knowledgebase.EdgeDetails
    EdgeData = knowledgebase.EdgeData
)
```

Reply:
yeah i think thats a solid idea, i can do that as i work on api gw